### PR TITLE
Add companies house api

### PR DESCRIPTION
**Pull request by kevinetienne**

### CODE REVIEW

File: web/company/companieshouse.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class CompaniesHouseException(Exception):
+    pass
+
+
+def api(query):
+    headers = {"Authorization": f"Basic {TOKEN}"}
+    response = requests.get(URL.format(query=query), headers=headers)
+    if response.status_code != 200:
+        logger.error(f"Company house responded with {response.status_code} - {response.text}")
+        msg = "Invalid response from companies house"
```
Comment (rossmiller) [inline web/company/companieshouse.py:25]: Not sure you need the `msg` var here

File: web/company/companieshouse.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+URL = "https://api.companieshouse.gov.uk/search/companies?q={query}&items_per_page=20"
+TOKEN = base64.b64encode(bytes(settings.COMPANIES_HOUSE_TOKEN, "utf-8")).decode("utf-8")
+
+
+class CompaniesHouseException(Exception):
+    pass
+
+
+def api(query):
```
Comment (rossmiller) [inline web/company/companieshouse.py:20]: Give this a more the function and argument a more descriptive name - `get(query_string)` or `api_get(query_string)` or something similar.

File: web/tests/company/test_companieshouse.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+from unittest.mock import Mock, patch
+
+import pytest
+
+from web.company.companieshouse import CompaniesHouseException, api
+
+
+@patch("web.company.companieshouse.requests.get")
+def test_api_raise_error(_):
```
Comment (rossmiller) [inline web/tests/company/test_companieshouse.py:9]: "test_api_raises_error"

**Review by rossmiller — CHANGES_REQUESTED** [review]
Some very minor changes.

File: web/company/companieshouse.py (hunk 2 above)
Comment (kevinetienne) [reply web/company/companieshouse.py]: called it `api_get_companies`

**Review by rossmiller — APPROVED** [review]
LGTM!

**Pull request merged**